web/core/components/issues/custom-properties.tsx

```
@@ -127,14 +129,49 @@
+          onBlur={handleBlur}
+          autoFocus
+          placeholder={`Add ${property.key}`}
+          className="text-sm w-full border rounded px-2 py-1 focus:outline-none focus:ring-2 focus:ring-custom-primary-100"
+        />
+      ),
+      boolean: (
+        <select
+          value={value}
+          onChange={handleChange}
+          onBlur={handleBlur}
+          autoFocus
```

Biome (1.9.4)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)